fastlane/lib/fastlane/actions/mailgun.rb

```diff
@@ -114,6 +119,13 @@ def self.mailgunit(options)
         unless options[:reply_to].nil?
           params.store(:"h:Reply-To", options[:reply_to])
         end
+
+        unless options[:attachment].nil?
+          attachment_filenames = [*options[:attachment]]
+          attachments = attachment_filenames.map { |filename| File.new(filename, 'rb') }
```

Comment:
its "read-binary", i think they do it this way in the documentation as if you don't use "rb" you get in troubles with non character files.